go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -112,11 +112,12 @@ func (qre *QueryExecutor) Execute() (reply *sqltypes.Result, err error) {
 		}
 
 		if reply == nil {
-			qre.tsv.qe.AddStats(planName, tableName, 1, duration, mysqlTime, 0, 1)
+			qre.tsv.qe.AddStats(planName, tableName, 1, duration, mysqlTime, 0, 0, 1)
 			qre.plan.AddStats(1, duration, mysqlTime, 0, 0, 1)
 			return
 		}
-		qre.tsv.qe.AddStats(planName, tableName, 1, duration, mysqlTime, int64(reply.RowsAffected), 0)
+
+		qre.tsv.qe.AddStats(planName, tableName, 1, duration, mysqlTime, int64(len(reply.Rows)), int64(reply.RowsAffected), 0)
```

Comment:
Since we're changing the existing metric from `reply.RowsAffected` to `reply.Rows`, will this mean that all `DELETE` plans will now produce zeroes instead of previously producing the amount of rows that were deleted?

Reply:
We'll rename QueryRowCounts to QueryRowsReturned (or something like that) and **exclude** irrelevant plan types.
So there will be no entries for Delete in that metric at all. And we'll stop emitting Select type metrics in the RowsAffected metric.

Reply:
when we discussed this earlier today, @harshit-gangal mentioned that metrics have been changed in the past without deprecation. I believe though that is was only changes to plan type names - a dimension of the metric, not the metric name itself.
To make it easier for users to upgrade let us add a new metric called `QueryRowsReturned` and deprecate the old one. We can schedule it for deletion in 15.0